projects/desktop/src/index.tsx

```diff
@@ -37,19 +38,39 @@ i18n
     },
   })
 
+const Loader = <Fragment />
+
 // Initiating redux store
 const store = configureStore()
 
+// This component will read the hydrated theme from redux
+const mapStateToProps = (state: IThemeState): IThemePayload => ({mode: state.themeReducer.mode})
```

Comment:
Here what you can do is :
```
const mapStateToProps = ({
  themeReducer: {mode},
}: IThemeState): IStoreProps => ({
mode,
})
```
It is better `IStoreProps` than `IThemePayload` for me